.travis.yml

```diff
@@ -12,6 +12,8 @@ stages:
   - name: build
     if: (branch = master AND type != pull_request)
         OR commit_message =~ /.*\[run ci\]/
+  - name: build_beta
+    if: branch = master
```

Comment:
This stage is declared but not used. Be more careful with merges and _do_ final check before review.